build.sh

```diff
@@ -200,13 +201,13 @@ while test "$1" != ""; do
 done
 
 # Suck in local vars
-. ${BUILDER_TOOLS}/builder_defaults.sh
+. "${BUILDER_TOOLS}"/builder_defaults.sh
```

Comment:
Since BUILDER_ROOT is mandatory and BUILDER_TOOLS is BUILDER_ROOT/tools, it will never be empty, these extra quotes are unnecessary